.flake8

```diff
@@ -20,17 +20,7 @@ per-file-ignores =
     # F841 local variable assigned to but never used
     # E741 ambiguous variable name 'l'
     __init__.py: F401, F403
-    man/build_check_rest.py: F403, F405
-    man/build_full_index_rest.py: F403, F405
-    man/parser_standard_options.py: F403, F405
-    man/build_class.py: F403, F405
-    man/build_class_rest.py: F403, F405
-    man/build_check.py: F403, F405
-    man/build_full_index.py: F403, F405
-    man/build_index.py: F403, F405
-    man/build_index_rest.py: F403, F405
-    man/build_keywords.py: F403, F405, E722
-    man/build_topics.py: F403, F405, E722
+    lib/init/grass.py: E722, F821, F841
```

Comment:
I don't think you intended to add this back when solving the conflicts

Reply:
Yeah, I must've missed it while updating the conflict. I've updated it now